### PR TITLE
Additional cleanup for PHPCS

### PR DESCRIPTION
**Pull request by davisshaver**

### CODE REVIEW

File: limit-modified-date.php [hunk 1]
```diff
@@ -11,22 +11,47 @@
...
+	 */
+	private $asset_version = '1.0';
 
-	function __construct() {
+	public function __construct() {
+		$this->nonce_key = $this->meta_key . '_nonce';
 
 		// Use original modified date
-		add_action('wp_insert_post_data', array( $this, 'use_original_modified_date' ), 20, 2 );
+		add_action( 'wp_insert_post_data', [ $this, 'use_original_modified_date' ], 20, 2 );
```
Comment (davisshaver) [inline limit-modified-date.php]: @billerickson One minor question that came up during code review - is there a specific reason why this is set as priority `20`? If so maybe we can document it here

**Comment by davisshaver** [comment]
@billerickson Open to merging this?

File: limit-modified-date.php (hunk 1 above)
Comment (billerickson) [reply limit-modified-date.php]: I wish I documented it at the time. I don't recall the reason, but I changed it from 10 to 20 so I think there's a reason. https://github.com/billerickson/Limit-Modified-Date/commit/71a356dcec391d7a5bcc558038cd690d0fe9955b

**Comment by billerickson** [comment]
Sorry for the delay. Just reviewed the changes and they look great. Merging now, although I won't push out an update on WordPress.org until there's a feature or improvement to include in the update.

**Pull request merged**